scripts/requirements.txt

```diff
@@ -7,3 +7,4 @@ docutils
 bibtexparser
 ghp-import
 wheel
+pygithub
```

Comment:
That's already in there../.

Reply:
Probably is missing a `pip install -r scripts/requirements.txt` in the `build_docker_image` job?